### ISSUE TITLE
Websocket (wss) connections stopped working for Firefox since v2.4.1

### ISSUE THREAD

**Issue by regmebaby** [traefik/traefik#7853]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?

I updated Traefik version in my docker-compose file from v2.4.0 to v2.4.1.

I use Traefik do expose GitLab-Omnibus instance (GitLab, Mattermost, GitLab Container Registry) and other containers.

I use HTTPS via Traefik / Let's Encrypt.

Mattermost uses Websocket (wss://mattermost.mydomain.tld/api/v4/websocket)

With Traefik version v2.4.0 everything worked well with Firefox and Chrome.

With Traefik version v2.4.1 it does not work with Firefox but it still works with Chrome.

I switched back to Traefik version v2.4.0 and it worked again with both browsers.

I switched again to Traefik version v2.4.1 and it does not work with Firefox but it still works with Chrome.

So the problems seems to be related with changes introduced in v2.4.1.

I have no clue why it still works with Chrome but no longer with Firefox.

Here is a Firefox-DevTools Network output captured with Traefik v2.4.0 and v2.4.1

![image](https://user-images.githubusercontent.com/8546924/106641779-e1e7fe80-6587-11eb-8da8-d87f1248ec47.png)

### What did you expect to see?

I expected to see Websockets working with Firefox. (Successful "101 Switching Protocols")

### What did you see instead?

Websockets no longer working with Firefox (Request stays open/uncompleted, NO "101 Switching Protocols")

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      2.4.1
Codename:     livarot
Go version:   go1.15.7
Built:        2021-02-01T16:28:12Z
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```yml
api:
  dashboard: true
entryPoints:
  web:
    address: ":80"
  websecure:
    address: ":443"
providers:
  docker:
    endpoint: "unix:///var/run/docker.sock"
    exposedByDefault: false
    network: traefik-net

certificatesResolvers:
  http:
    acme:
      email: foo@mydomain.de
      storage: acme.json
      httpChallenge:
        entryPoint: web

#accessLog:
#  format: common

log:
  level: DEBUG
```

### If applicable, please paste the log output in DEBUG level (`--log.level=DEBUG` switch)

With v2.4.0:
```
traefik             | time="2021-02-02T17:43:06Z" level=debug msg="vulcand/oxy/roundrobin/rr: begin ServeHttp on request" Request="{\"Method\":\"GET\",\"URL\":{\"Scheme\":\"\",\"Opaque\":\"\",\"User\":null,\"Host\":\"\",\"Path\":\"/api/v4/websocket\",\"RawPath\":\"\",\"ForceQuery\":false,\"RawQuery\":\"\",\"Fragment\":\"\",\"RawFragment\":\"\"},\"Proto\":\"HTTP/1.1\",\"ProtoMajor\":1,\"ProtoMinor\":1,\"Header\":{\"Accept\":[\"*/*\"],\"Accept-Encoding\":[\"gzip, deflate, br\"],\"Accept-Language\":[\"de-DE,de;q=0.8,en-US;q=0.5,en;q=0.3\"],\"Cache-Control\":[\"no-cache\"],\"Connection\":[\"keep-alive, Upgrade\"],\"Cookie\":[\"rl_anonymous_id=REDACTED; rl_user_id=REDACTED; _ga=GA1.2.1812274325.1612286540; _gid=GA1.2.1859186051.1612286540; MMAUTHTOKEN=REDACTED; MMUSERID=REDACTED; MMCSRF=REDACTED\"],\"Origin\":[\"https://mattermost.mydomain.de\"],\"Pragma\":[\"no-cache\"],\"Sec-Websocket-Extensions\":[\"permessage-deflate\"],\"Sec-Websocket-Key\":[\"REDACTED\"],\"Sec-Websocket-Version\":[\"13\"],\"Upgrade\":[\"websocket\"],\"User-Agent\":[\"Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:78.0) Gecko/20100101 Firefox/78.0\"],\"X-Forwarded-Host\":[\"mattermost.mydomain.de\"],\"X-Forwarded-Port\":[\"443\"],\"X-Forwarded-Proto\":[\"wss\"],\"X-Forwarded-Server\":[\"b5303fb7c979\"],\"X-Real-Ip\":[\"REDACTED\"]},\"ContentLength\":0,\"TransferEncoding\":null,\"Host\":\"mattermost.mydomain.de\",\"Form\":null,\"PostForm\":null,\"MultipartForm\":null,\"Trailer\":null,\"RemoteAddr\":\"REDACTED\",\"RequestURI\":\"/api/v4/websocket\",\"TLS\":null}"
traefik             | time="2021-02-02T17:43:06Z" level=debug msg="vulcand/oxy/roundrobin/rr: Forwarding this request to URL" ForwardURL="http://172.22.0.5:80" Request="{\"Method\":\"GET\",\"URL\":{\"Scheme\":\"\",\"Opaque\":\"\",\"User\":null,\"Host\":\"\",\"Path\":\"/api/v4/websocket\",\"RawPath\":\"\",\"ForceQuery\":false,\"RawQuery\":\"\",\"Fragment\":\"\",\"RawFragment\":\"\"},\"Proto\":\"HTTP/1.1\",\"ProtoMajor\":1,\"ProtoMinor\":1,\"Header\":{\"Accept\":[\"*/*\"],\"Accept-Encoding\":[\"gzip, deflate, br\"],\"Accept-Language\":[\"de-DE,de;q=0.8,en-US;q=0.5,en;q=0.3\"],\"Cache-Control\":[\"no-cache\"],\"Connection\":[\"keep-alive, Upgrade\"],\"Cookie\":[\"rl_anonymous_id=REDACTED; rl_user_id=REDACTED; _ga=GA1.2.1812274325.1612286540; _gid=GA1.2.1859186051.1612286540; MMAUTHTOKEN=REDACTED; MMUSERID=REDACTED; MMCSRF=REDACTED\"],\"Origin\":[\"https://mattermost.mydomain.de\"],\"Pragma\":[\"no-cache\"],\"Sec-Websocket-Extensions\":[\"permessage-deflate\"],\"Sec-Websocket-Key\":[\"REDACTED\"],\"Sec-Websocket-Version\":[\"13\"],\"Upgrade\":[\"websocket\"],\"User-Agent\":[\"Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:78.0) Gecko/20100101 Firefox/78.0\"],\"X-Forwarded-Host\":[\"mattermost.mydomain.de\"],\"X-Forwarded-Port\":[\"443\"],\"X-Forwarded-Proto\":[\"wss\"],\"X-Forwarded-Server\":[\"b5303fb7c979\"],\"X-Real-Ip\":[\"REDACTED\"]},\"ContentLength\":0,\"TransferEncoding\":null,\"Host\":\"mattermost.mydomain.de\",\"Form\":null,\"PostForm\":null,\"MultipartForm\":null,\"Trailer\":null,\"RemoteAddr\":\"REDACTED\",\"RequestURI\":\"/api/v4/websocket\",\"TLS\":null}"
```
With v2.4.1

``` 
no log entries for websocket and no exceptions
```

**Comment by ldez** [comment]
Fixed by #7847

**Comment by regmebaby** [comment]
> 
> 
> Fixed by #7847

I can confirm that it is fixed in v2.4.2, thanks! 🙂
